client/config/crd/groupsnapshot.storage.k8s.io_volumegroupsnapshotclasses.yaml

```diff
@@ -3,9 +3,9 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.3
     api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/814"
   creationTimestamp: null
+    controller-gen.kubebuilder.io/version: v0.12.0
```

Comment:
This was moved from the `annotations` section to here, that looks wrong. The same happened in the other files.